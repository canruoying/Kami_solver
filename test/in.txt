1112212211
1122111221
1221131122
2211333112
2113313311
2133121331
2113313311
2211333112
1221131122
1122111221
1112212211
3111222111
3311121113
1331111133
1133111331
1113313311