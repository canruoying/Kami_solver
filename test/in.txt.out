=======
1 1 1 2 2 1 2 2 1 1 
1 1 2 2 1 1 1 2 2 1 
1 2 2 1 1 3 1 1 2 2 
2 2 1 1 3 3 3 1 1 2 
2 1 1 3 3 1 3 3 1 1 
2 1 3 3 1 2 1 3 3 1 
2 1 1 3 3 1 3 3 1 1 
2 2 1 1 3 3 3 1 1 2 
1 2 2 1 1 3 1 1 2 2 
1 1 2 2 1 1 1 2 2 1 
1 1 1 2 2 1 2 2 1 1 
3 1 1 1 2 2 2 1 1 1 
3 3 1 1 1 2 1 1 1 3 
1 3 3 1 1 1 1 1 3 3 
1 1 3 3 1 1 1 3 3 1 
1 1 1 3 3 1 3 3 1 1 
=======
 ||
 ||
 || Step 1
\||/
 \/
=======
1 1 1 1 1 1 2 2 1 1 
1 1 1 1 1 1 1 2 2 1 
1 1 1 1 1 3 1 1 2 2 
1 1 1 1 3 3 3 1 1 2 
1 1 1 3 3 1 3 3 1 1 
1 1 3 3 1 2 1 3 3 1 
1 1 1 3 3 1 3 3 1 1 
1 1 1 1 3 3 3 1 1 1 
1 1 1 1 1 3 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
3 1 1 1 1 1 1 1 1 1 
3 3 1 1 1 1 1 1 1 3 
1 3 3 1 1 1 1 1 3 3 
1 1 3 3 1 1 1 3 3 1 
1 1 1 3 3 1 3 3 1 1 
=======
 ||
 ||
 || Step 2
\||/
 \/
=======
2 2 2 2 2 2 2 2 1 1 
2 2 2 2 2 2 2 2 2 1 
2 2 2 2 2 3 2 2 2 2 
2 2 2 2 3 3 3 2 2 2 
2 2 2 3 3 1 3 3 2 2 
2 2 3 3 1 2 1 3 3 2 
2 2 2 3 3 1 3 3 2 2 
2 2 2 2 3 3 3 2 2 2 
2 2 2 2 2 3 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
3 2 2 2 2 2 2 2 2 2 
3 3 2 2 2 2 2 2 2 3 
1 3 3 2 2 2 2 2 3 3 
1 1 3 3 2 2 2 3 3 1 
1 1 1 3 3 2 3 3 1 1 
=======
 ||
 ||
 || Step 3
\||/
 \/
=======
3 3 3 3 3 3 3 3 1 1 
3 3 3 3 3 3 3 3 3 1 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 1 3 3 3 3 
3 3 3 3 1 2 1 3 3 3 
3 3 3 3 3 1 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
3 3 3 3 3 3 3 3 3 3 
1 3 3 3 3 3 3 3 3 3 
1 1 3 3 3 3 3 3 3 1 
1 1 1 3 3 3 3 3 1 1 
=======
 ||
 ||
 || Step 4
\||/
 \/
=======
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 2 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
=======
 ||
 ||
 || Step 5
\||/
 \/
=======
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
=======
Time Elapsed: 266ms
